parallel_cluster/bin/sync_config_files.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
-aws s3 sync cromwell/ s3://umccr-temp-dev/Alexis_parallel_cluster_test/cromwell/
-aws s3 sync bootstrap/ s3://umccr-temp-dev/Alexis_parallel_cluster_test/bootstrap/
-aws s3 sync bcbio s3://umccr-temp-dev/Alexis_parallel_cluster_test/bcbio/
\ No newline at end of file
+ROOT_PATH="umccr-temp-dev/Alexis_parallel_cluster_test"
```

Comment:
Might want to move those config files to the research bucket? Otherwise they'll be gone soon :-S

Reply:
All of these files are on github. Do you have a suggested folder for them to go. Will mean I have to do fix on the pre_install and post_install args that pull from s3.

Reply:
Created a root env var so will be ready to change quickly.